profiles/views.py

```diff
@@ -3,22 +3,24 @@
 from django.db.models import Count
 from django.http import HttpRequest, HttpResponse
 from django.shortcuts import get_object_or_404, redirect, render
-from django.urls import reverse_lazy
 from django.utils.translation import gettext_lazy as _
 from django.views import View
 
 from comments.models import Comment
 from posts.models import Post
 from users.models import ForumUser
 
-from .forms import EditProfileForm
+from .forms import EditProfileForm, EditProfileLinksForm
+from .models import Profile
 
 
 class ProfileView(LoginRequiredMixin, View):
     template_name = "profiles/profile.html"
 
     def get(self, request: HttpRequest, profile: str) -> HttpResponse:
         user = get_object_or_404(ForumUser, username=profile)
+        Profile.objects.get_or_create(user=self.request.user)
+        profile = get_object_or_404(Profile, user=self.request.user)
```

Comment:
As I remember first value will be True or False, and second is profile
```suggestion
        _, profile = Profile.objects.get_or_create(user=self.request.user)
```